src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -213,6 +213,7 @@ Map<String, BigDecimal> getWinnerToFractionalValue() {
   // it is active or not.
   enum StatusForRound {
     ACTIVE,
+    LOCKED_IN,
```

Comment:
I would update the comment above to be more explicit about what this new status value means. In particular, the comment above ("StatusForRound represents the ballot's status on a given round") is confusing because this new status value might apply to only a portion of a given ballot.

More generally: if I'm not mistaken, all of the existing `StatusForRound` values have been used to refer to an entire CVR, so this new status is both introducing a new type of status and changing the meaning of the other statuses. That's potentially fine, but I'm wondering whether there are dependencies that we need to update accordingly. E.g. does the behavior in `recordSelectionForCastVoteRecord` still make complete sense? At a minimum, I imagine that we need to add some clarifying comments in various places and/or rename some things.